int16 flag
float64 time
float64 forward_velocity
float64 angular_veolocity
int16 subject
float64 range
float64 bearing